src/mug/tls.gleam

```diff
@@ -0,0 +1,630 @@
+import gleam/bytes_builder.{type BytesBuilder}
```

Comment:
This module seems to be almost exactly the same as the existing module. We could merge them into one rather than having duplicate functions for the programmer to pick between.